firestore/app/src/main/java/com/google/example/firestore/DocSnippets.java

```diff
@@ -1361,4 +1361,13 @@ public void onComplete(@NonNull Task<Void> task) {}
         });
         // [END update_delete_field]
     }
+
+    public void countAggregateQuery() {
+        // [START count_aggregate_query]
+        Query query = db.collection("games/chess/players").whereEqualTo("online", true);
+        AggregateQuery countQuery = query.count();
+        AggregateQuerySnapshot snapshot = countQuery.get(AggregateSource.SERVER).getResult();
+        Log.d(TAG, "Count: " + snapshot.getCount());
```

Comment:
IIRC `get()` returns a `Task` - it might be better to attach a listener instead, just like we do for all the other Firestore queries:

```suggestion
        countQuery.get(AggregateSource.SERVER).addOnCompleteListener(new OnCompleteListener<AggregateQuerySnapshot>() {
    @Override
    public void onComplete(@NonNull Task<AggregateQuerySnapshot> task) {
        if (task.isSuccessful()) {
            // Count fetched successfully
            AggregateQuerySnapshot snapshot = task.getResult();
            Log.d(TAG, "Count: " + snapshot.getCount());
        } else {
            Log.d(TAG, "Count failed: ", task.getException());
        }
    }
});;
```